traits/category.py

```diff
@@ -88,6 +92,8 @@ def add_traits_meta_data ( self, bases, class_dict, base_traits,
 #  'Category' class:
 #-------------------------------------------------------------------------------
 
+
+@six.add_metaclass(MetaCategory)
```

Comment:
Would this look neater using `with_metaclass`?  `class Category(with_metaclass(MetaCategory, object))`?